concourse-ci/azuredeploy.json

```diff
@@ -0,0 +1,284 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
+      "type": "string",
+      "defaultValue": "vnet",
+      "metadata": {
+        "description": "name of the virtual network"
+      }
+    },
+    "virtualNetworkAddressPrefix": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/16",
+      "metadata": {
+        "description": "address prefix of the virtual network"
+      }
+    },
+    "subnetNameForBosh": {
+      "type": "string",
+      "defaultValue": "Bosh",
+      "metadata": {
+        "description": "name of the subnet for Bosh"
+      }
+    },
+    "subnetAddressPrefixForBosh": {
+      "type": "string",
+      "defaultValue": "10.0.0.0/24",
+      "metadata": {
+        "description": "address prefix of the subnet for Bosh"
+      }
+    },
+    "subnetNameForConcourse": {
+      "type": "string",
+      "defaultValue": "Concourse",
+      "metadata": {
+        "description": "name of the subnet for Concourse"
+      }
+    },
+    "subnetAddressPrefixForConcourse": {
+      "type": "string",
+      "defaultValue": "10.0.16.0/24",
+      "metadata": {
+        "description": "address prefix of the subnet for Concourse"
+      }
```

Comment:
Usually things that don't need to be globablly unique should become variables. Users can change these values in "Deploy to Azure" experience in the Portal before deploying.